doc/webservices/iap.rst

```diff
@@ -1,10 +1,41 @@
 :types: api
 
-
-:code-column:
-
 .. _webservices/iap:
 
+.. using sphinx-patchqueue:
```

Comment:
is it a doc for those who wrote doc? 😉

Reply:
Yep, though maybe it should be in a doc/README file instead?